### PR TITLE
[zh]Update content/zh/docs/reference/labels-annotations-taints/audit-annotations.md

### PR DESCRIPTION
**Pull request by kinzhi**

### CODE REVIEW

File: content/zh/docs/reference/labels-annotations-taints/audit-annotations.md [hunk 1]
```diff
@@ -99,4 +99,38 @@ PodSecurity "restricted:latest": allowPrivilegeEscalation != false (container
...
+
+This annotation indicates whether or not a request was authorized in Kubernetes audit logs.
+
+See [Auditing](/docs/tasks/debug-application-cluster/audit/) for more information.
+-->
+## authorization.k8s.io/decision {#authorization-k8s-io-decision}
+
+例子：`authorization.k8s.io/decision: "forbid"`
+
+此注解表示请求是否在 Kubernetes 审计日志中获得授权。
```
Comment (tengqm) [inline content/zh/docs/reference/labels-annotations-taints/audit-annotations.md:117]:
```suggestion
此注解在 Kubernetes 审计日志中表示请求是否获得授权。
```

File: content/zh/docs/reference/labels-annotations-taints/audit-annotations.md [hunk 2]
```diff
@@ -99,4 +99,38 @@ PodSecurity "restricted:latest": allowPrivilegeEscalation != false (container
...
+
+有关详细信息，请参阅 [审计](/zh/docs/tasks/debug-application-cluster/audit/)。
```
Comment (tengqm) [inline content/zh/docs/reference/labels-annotations-taints/audit-annotations.md:119]:
```suggestion
有关详细信息，请参阅[审计](/zh/docs/tasks/debug-application-cluster/audit/)。
```

File: content/zh/docs/reference/labels-annotations-taints/audit-annotations.md [hunk 3]
```diff
@@ -99,4 +99,38 @@ PodSecurity "restricted:latest": allowPrivilegeEscalation != false (container
...
+
+See [Auditing](/docs/tasks/debug-application-cluster/audit/) for more information.
+-->
+## authorization.k8s.io/reason {#authorization-k8s-io-reason}
+
+例子：`authorization.k8s.io/reason: "Human-readable reason for the decision"`
+
+此注解给出了 Kubernetes 审计日志中 [decision](#authorization-k8s-io-decision) 的原因。
+
+有关详细信息，请参阅 [审计](/zh/docs/tasks/debug-application-cluster/audit/)。
```
Comment (tengqm) [inline content/zh/docs/reference/labels-annotations-taints/audit-annotations.md:136]:
```suggestion
有关详细信息，请参阅[审计](/zh/docs/tasks/debug-application-cluster/audit/)。
```

**Comment by mengjiao-liu** [comment]
/lgtm
/approve
Thanks!

**Pull request merged**